airflow-core/src/airflow/ui/src/utils/hitl.ts

```diff
@@ -167,10 +167,23 @@ export const getHITLFormData = (paramsDict: ParamsSpec, option?: string): HITLRe
 };
 
 export const getHITLState = (translate: TFunction, hitlDetail: HITLDetail) => {
-  const { chosen_options: chosenOptions, options, params, response_received: responseReceived } = hitlDetail;
+  const {
+    chosen_options: chosenOptions,
+    options,
+    params,
+    response_received: responseReceived,
+    task_instance: { state: taskInstanceState },
+  } = hitlDetail;
+
+  const isNotDeferred = taskInstanceState !== "deferred";
 
   let stateType: [string, string] = ["responseRequired", "responseReceived"];
 
+  if (!responseReceived && isNotDeferred) {
+    // need to update this after unfreezing hitl.json
```

Comment:
```suggestion
    // TODO: update this after unfreezing hitl.json
```

editor normally can check this

Reply:
Thanks! good suggestion.